persistence/src/main/scala/hmda/persistence/demo/DemoData.scala

```diff
@@ -85,6 +85,7 @@ object DemoData {
       case (instId: String, filings: Seq[Filing]) =>
         val filingActor = system.actorOf(FilingPersistence.props(instId))
         filings.foreach { filing => filingActor ? CreateFiling(filing) }
+        Thread.sleep(100)
```

Comment:
Yeah, I kept getting failures in `SubmissionParseErrorsPathsSpec` on my branch, until I added the `Thread.sleep`. I'm as puzzled as you 😕 

I added `Thread.sleep` out of superstition, and it hasn't failed since. I can remove it again, to see whether the failures continue.